django/db/backends/postgresql/introspection.py

```diff
@@ -191,16 +191,21 @@ def get_constraints(self, cursor, table_name):
                 "foreign_key": tuple(used_cols.split(".", 1)) if kind == "f" else None,
                 "check": kind == "c",
                 "index": False,
+                "definition": None,
             }
         # Now get indexes
         cursor.execute("""
             SELECT
                 indexname, array_agg(attname), indisunique, indisprimary,
-                array_agg(ordering), amname
+                array_agg(ordering), amname, exprdef
             FROM (
                 SELECT
                     c2.relname as indexname, idx.*, attr.attname, am.amname,
                     CASE
+                        WHEN idx.indexprs IS NOT NULL THEN
```

Comment:
Sorry, I missed this PR.
From what I found [in the docs](https://www.postgresql.org/docs/9.5/static/catalog-pg-index.html), this looks like the right thing to do. Thanks :+1: